packages/tools/Makefile

```
@@ -203,13 +230,29 @@ tools-install:
 docker-install: ${CONFIG_DOCKER_FILE} docker-config-proxy
 
 docker-config-proxy:
```

checkmake (0.3.2)

[warning] Target body for "docker-config-proxy" exceeds allowed length of 5 lines (25).

(maxbodylength)